operators/A41/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "A41",
+    "website": "https://www.a41.io",
+    "description": "A41 is a blockchain infra-service provider that validates and paves paths never ventured before.",
+    "logo": "https://github.com/a41-official/metadata/blob/main/logo.png",
```

Comment:
@darron1217 this needs to be a raw github url - please check any other operator to see how to should look like. thanks!

Reply:
Updated. Thanks for the feedback :)